### PR TITLE
add ZAP Chat Video 14 subtitles

### PR DESCRIPTION
**Pull request by 8linkz**
It's that time again :-)

Video url: https://www.youtube.com/watch?v=Cm_ol-PA738

### CODE REVIEW

File: subtitles/ZAP Chat/ZAP_Chat_14_Google_Summer_of_Code.srt [hunk 1]
```diff
@@ -0,0 +1,1659 @@
...
+00:20:42,920 --> 00:20:43,300
+
+
+291
+00:20:43,300 --> 00:20:45,420
+Make some scanning as well.
+
+292
+00:20:45,420 --> 00:20:53,680
+I just don't see the web-based UI being there, you know, for the red teamers out there or the, well, I don't know.
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_14_Google_Summer_of_Code.srt]: `teams`

File: subtitles/ZAP Chat/ZAP_Chat_14_Google_Summer_of_Code.srt [hunk 2]
```diff
@@ -0,0 +1,1659 @@
...
+00:20:19,040 --> 00:20:19,540
+Absolutely.
+
+285
+00:20:19,540 --> 00:20:24,420
+So you have, in essence, you've got a couple of personas that are meant, you know, for the web-based UI.
+
+286
+00:20:24,420 --> 00:20:31,500
+Anything that's CI/CD, operational, DevSecOpsy, people are used to in operation rooms, right?
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_14_Google_Summer_of_Code.srt]: `DevSecOps`

File: subtitles/ZAP Chat/ZAP_Chat_14_Google_Summer_of_Code.srt [hunk 3]
```diff
@@ -0,0 +1,1659 @@
...
+00:09:46,760 --> 00:09:51,680
+was actually a suggestion to create a Chrome add-on
+
+116
+00:09:51,680 --> 00:09:54,020
+to add what was effectively a heads-up display.
+
+117
+00:09:54,020 --> 00:09:59,060
+Now, this was suggested by a student called David Scribonia,
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_14_Google_Summer_of_Code.srt]: Scrobonia

File: subtitles/ZAP Chat/ZAP_Chat_14_Google_Summer_of_Code.srt [hunk 4]
```diff
@@ -0,0 +1,1659 @@
...
+00:05:25,800 --> 00:05:31,860
+designed to another OWASP project but it turned out that was a very small shim around another
+
+60
+00:05:31,860 --> 00:05:38,140
+project which wasn't OWASP project so in the end it got transferred to ZAP, because that was
+
+61
+00:05:38,140 --> 00:05:43,740
+the Ajax spider and it was a wrapper around crawl jacks. So the Ajax spider came out of google summer
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_14_Google_Summer_of_Code.srt]: `crawl jacks` → `crawljax`

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]
Thanks!

**Pull request merged**